packages/contracts/test/voting/admin.ts

```diff
@@ -184,7 +175,7 @@ describe('Admin plugin', function () {
       const expectedDummyResults = ['0x'];
 
       await expect(await plugin.executeProposal(dummyMetadata, dummyActions))
```

Comment:
Is `await plugin....` still necessary?

Reply:
Fixed in 31773140e64edfb0d2ba37bb16d6defd856d0577